readme.txt

```diff
@@ -22,7 +22,7 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
-= Unreleased
+= 2.3.1 =
 * Hide free trial plan picker banner when viewing iframe #1404
```

Comment:
This is already included in 2.3.0. 😬

Reply:
Ah, I see. Will fix that

Reply:
@chihsuan Fixed it in [61215c8](https://github.com/Automattic/wc-calypso-bridge/pull/1418/commits/61215c845a00904e553798f2edecf6b41f2725fe), there was copy pasta issue